Bychynski_Pavel_HW_4.py

```diff
@@ -0,0 +1,212 @@
+# # CW_test
+# def mass (num):
+#     print("lul", num)
+# num = 2
+# mass(4)
+# print(num)
+#
+# def dt (a=1, b=2):
+#     mass(999)
+#     print("line:", a, b)
+# dt("check")
+#
+# def mult (f=1, s=1):
+#     print("mult: ", f, "*", s, "=", f * s)
+# t_n1 = int(input("1"))
+# t_n2 = int(input("2"))
+# mult(t_n1, t_n2)
+#
+# # 4.1
+# # a bunch of defines for calculations
+# def add ():
+#     print(user_num_1, "+", user_num_2, "=", user_num_1 + user_num_2)
+# def sub ():
+#     print(user_num_1, "-", user_num_2, "=", user_num_1 - user_num_2)
+# def mult ():
+#     print(user_num_1, "*", user_num_2, "=", user_num_1 * user_num_2)
+# def div ():
+#     print(user_num_1, "/", user_num_2, "=", user_num_1 / user_num_2)
+# def mod ():
+#     print(user_num_1, "%", user_num_2, "=", user_num_1 % user_num_2)
+# def exp ():
+#     print(user_num_1, "**", user_num_2, "=", user_num_1 ** user_num_2)
+# def fdiv ():
+#     print(user_num_1, "//", user_num_2, "=", user_num_1 // user_num_2)
+# # almost identical to previous calculatiors
+# def maxi ():
+#     minmax_l = []
+#     i_numb = 1
+#     minmax_numb = int(input("input amount of numbers (>1): "))
+#     if minmax_numb > 1:
+#         while i_numb <= minmax_numb:
+#             num_e = int(input("input your number: "))
+#             i_numb += 1
+#             minmax_l.append(num_e)
+#         print("max =", max(minmax_l))
+#     else:
+#         print("not enough elements")
+# def mini ():
+#     minmax_l = []
+#     i_numb = 1
+#     minmax_numb = int(input("input amount of numbers (>1): "))
+#     if minmax_numb > 1:
+#         while i_numb <= minmax_numb:
+#             num_e = int(input("input your number: "))
+#             i_numb += 1
+#             minmax_l.append(num_e)
+#         print("min =", min(minmax_l))
+#     else:
+#         print("not enough elements")
+#
+# print("That\'s a simple calculator")
+# print(
+#     """
+#     Type \'add\' for addition, \'sub\' for subtraction,
+#     \'mult\' for multiplication, \'div\' for division,
+#     \'mod\' for modulus, \'exp\' for exponentiation, \'fdiv\' for floor division,
+#     \'max\' to find the maximum number or \'min\' to find the minimum number.
+#     Type \'exit\' to exit.
+# """)
+# oper = input()
+# if oper == "add" or oper == "sub" or oper == "mult" or oper == "div" or oper == "mod" or oper == "exp" or oper == "fdiv":
```

Comment:
Can be simplify
listOpers = ["add", "sub", "mult", "div", "mod","exp","fdiv"]
if oper in listOpers: